src/main/java/frc/robot/subsystems/Arm/ArmSubsystem.java

```diff
@@ -55,16 +55,25 @@ public void periodic() {
 
     pidOutput = MathUtil.clamp(pidOutput, -1, 1);
     armIO.setArmSpeed(pidOutput);
+
     NetworkTableInstance.getDefault().getEntry("/Arm/ArmAngle").setNumber(armIO.getArmEncoderAngle());
     NetworkTableInstance.getDefault().getEntry("/Arm/pidOutput").setNumber(pidOutput);
     NetworkTableInstance.getDefault().getEntry("/Arm/armGoal")
         .setNumber(Units.radiansToDegrees(pidController.getGoal().position));
     armIO.update();
+
+    if (armIO.isEncoderConnected()) {
+      armIO.setArmSpeed(pidOutput);
+    }
+    else {
+      armIO.setArmSpeed(0);
+    }
   }
 
   public Command spinArm(double targetAngle) {
     return this.runOnce(() -> {
       pidController.setGoal(Units.degreesToRadians(targetAngle));
-    }).andThen(Commands.waitUntil(() -> pidController.atGoal()));
+    }).andThen(Commands.waitUntil(() -> pidController.atGoal()))
+    .onlyIf(() -> armIO.isEncoderConnected());
```

Comment:
Again this escape needs to produce a warning. You could put the warning in the function being called. Since you need it every time the function is called.